CHANGELOG.md

```diff
@@ -1,3 +1,146 @@
+# 70.0.0
+
+This major change removes all property coding behavior from our components.
```

Comment:
Do you think this concludes the release fully? How about the PageControl and NavDrawer work?

Reply:
Nope. Fleshed out.